guides/developers-guide/concepts/tags.md

```diff
@@ -0,0 +1,47 @@
+
+Open Commerce does offer out-of-the-box functionality to build product hierarchies with tags and navigation trees, although it's important to understand that tags are specific on a product level; i.e. they do not apply down to individual variants of a specific product (or even options of variants).
+
+> NOTE: Tags are set on products, and not on variants Let's build a tagging hierarchy that looks like this:
+
+```sh
+          |-- Women  (top-level category)
+              |-- Shoes  (second-level category)
+                  |-- Sneakers  (third-level category)
+                  |   |-- Adides MaxRun 370 (product)
+                  |   |-- Nuke Sports+ (product)
+                  |
+                  |-- Heels  (third-level category)
+                  |   |-- Bior Classic 10cm  (product)
+                  |   |-- Loubootain Eleganza 6cm (product)
+```
+
+## 1. Create the tags in reaction-admin
+
+- To create a top tag (Women), login to reaction-admin and navigate to Tags. Click on the 'Create a tag' button.
+
+- Then, fill in the form, make sure to tick the "Tag is enabled in storefront" checkbox, and save your new tag.
+
+### Creating the second-level and third-level tags
+
+With Open Commerce, there is no hierarchy at the tag-level. You'll create your second-level and third-level tags all exactly like you created your top-level tag, and you'll organize them in a hierarchy in the last step using a navigation tree.
+
+When creating your second and third-level tags (or any deeper hierarchy for that matter), a good practice is to include the name of the "parent" tags in the "child" tag's name and slug. Because tag names and slugs need to be unique, you could run into conflicts if — for example — you also want to have a "Men" category with "Shoes" and "Sneakers" tags under it.
+
+To prevent any confusion or conflict, you would give the "Shoes" tag under "Women" the name and slug "women-shoes". The display name, however, stays "Shoes" in order not to overwhelm customers. For your "Sneakers" tag under "Shoes", itself under "Women", give it the "women-shoes-sneakers" tag name and slug.
+
+## 2. Assign tags to products
+
+After creating the tags (read: product categories), you need to tell the products which category they belong to. This can be set in the product settings for all products (not varient or options level). If you like your product to be found regardless if one browses a category that is a direct parent or an indirect child, you need to ensure that the product gets assigned all tag names up to the root tag assigned. For example, our "Adides MaxRun 370" product needs to be assigned the "women-shoes-sneakers" tag, but also "women-shoes" and "women".
```

Comment:
```suggestion
After creating the tags (read: product categories), you need to tell the products which category they belong to. This can be set in the product settings for all products (not variant or options level). If you like your product to be found regardless if one browses a category that is a direct parent or an indirect child, you need to ensure that the product gets assigned all tag names up to the root tag assigned. For example, our "Adides MaxRun 370" product needs to be assigned the "women-shoes-sneakers" tag, but also "women-shoes" and "women".
```